nav2_bringup/launch/nav2_params.yaml

```diff
@@ -39,14 +40,16 @@ global_costmap:
       scan:
         topic: /scan
         max_obstacle_height: 2.0
-        clearing: False
+        clearing: True
         marking: True
 robot_centric_costmap:
   robot_centric_costmap:
     ros__parameters:
       plugin_names: ["obstacle_layer", "inflation_layer"]
       plugin_types: ["nav2_costmap_2d::ObstacleLayer", "nav2_costmap_2d::InflationLayer"]
       global_frame: "base_link"
+      origin_x: -5.0
```

Comment:
how large is this local costmap? This origin should be -size/2 for the ray casting to work. a 10m local costmap would be a little excessive

Reply:
I was using for testing the default size which is 10x10 m. I'll change it on the yaml so it's something between 3 to 5 m with the origin, as you mention, at -size/2.